src/ui/public/fixed_scroll.js

```diff
@@ -98,16 +103,28 @@ uiModules
         listen();
       }
 
-      // reset when the width or scrollWidth of the $el changes
-      $scope.$watchMulti([
-        function () { return $el.prop('scrollWidth'); },
-        function () { return $el.width(); }
-      ], setup);
+      let width;
+      let scrollWidth;
+      let widthCheckTimeout = $timeout(checkWidth, 500);
```

Comment:
On the plus side, as long as the content of the page hasn't changed the timeout function is pretty lightweight. Definitely still open to other ideas though.